app/views/credit_mutations/index.html.slim

```diff
@@ -39,7 +39,7 @@
                     = link_to mutation.activity&.title, mutation.activity
                 td
                   = link_to mutation.created_by.name, mutation.created_by
-                td
+                td.text-nowrap
```

Comment:
Why nowrap? What happens if this is too long?

Reply:
See the screenshot in the main post